app/models/manageiq/providers/openstack/cloud_manager/provision_workflow.rb

```diff
@@ -91,4 +91,5 @@ def dialog_name_from_automate(message = 'get_dialog_name')
   def self.provider_model
     ManageIQ::Providers::Openstack::CloudManager
   end
+  private_class_method :provider_model
```

Comment:
I think we have to undo this one.  It probably is meant to be public.  cc @Ladas 

```
app/models/manageiq/providers/azure/cloud_manager/provision_workflow.rb:  def self.provider_model
app/models/manageiq/providers/azure/cloud_manager/provision_workflow.rb:  private_class_method :provider_model
app/models/manageiq/providers/google/cloud_manager/provision_workflow.rb:  def self.provider_model
app/models/manageiq/providers/google/cloud_manager/provision_workflow.rb:  private_class_method :provider_model
app/models/manageiq/providers/microsoft/infra_manager/provision_workflow.rb:  def self.provider_model
app/models/manageiq/providers/openstack/cloud_manager/provision_workflow.rb:  def self.provider_model
app/models/manageiq/providers/openstack/cloud_manager/provision_workflow.rb:  private_class_method :provider_model
app/models/manageiq/providers/redhat/infra_manager/provision_workflow.rb:  def self.provider_model
app/models/manageiq/providers/vmware/infra_manager/provision_workflow.rb:  def self.provider_model
app/models/miq_provision_virt_workflow.rb:    return ["vms.template = ? AND vms.ems_id IS NOT NULL", true] unless self.class.respond_to?(:provider_model)
app/models/miq_provision_virt_workflow.rb:    ["vms.template = ? AND vms.ems_id in (?)", true, self.class.provider_model.pluck(:id)]
```